template.yaml

```diff
@@ -161,7 +162,7 @@ Resources:
                   - cognito-idp:DescribeUserPoolDomain
                 Resource: "*"
 
-  TriggerExecutionRole:
+  CreateUsersRole:
```

Comment:
We do not use the term "trigger" anywhere in the code. The Role is now required by two functions, the trigger and the CreateUser REST API call